spring-integration-hazelcast/.gitignore

```diff
@@ -0,0 +1,4 @@
+/target/
+/.settings/
+/.classpath
+/.project
```

Comment:
Hi, @erenavsarogullari !

Let's start to work with your solution!

See my comments, and if you won't mind in changes there is no reason to comment like `Fixed`, do not spam me :smile: .
Please, add comment like `Pushed` when you have pushed some commit.

So, the first comment: this file is redundant. All these ignore options are covered in the root dir with the same `.gitignore`